calyx-py/test/builder_example.py

```diff
@@ -35,21 +49,34 @@ def build():
         # Directly use the ast.CompPort object `add_out`.
         # This is useful HACK when we haven't defined the cell yet but still want
         # to use its ports.
+        # ANCHOR: bare_use
         sum.in_ = add_out
+        # ANCHOR_END: bare_use
         compute_sum.done = sum.done
 
+    # ANCHOR: this
     # Use `this()` method to access the ports on the current component
     this = main.this()
+    # ANCHOR_END: this
+    # ANCHOR: continuous
     with main.continuous:
-        this.in_ = this.out
+        this.out = sum.out
+    # ANCHOR_END: continuous
 
-    main.control += [
-        update_operands,
-        compute_sum,
-    ]
+    # ANCHOR: control
+    main.control += [{update_operands, compute_sum}]
```

Comment:
to avoid non-determinism replace with 
```
main.control += par(update_operands, compute_sum)
```
if you want a wrapping seq, then put square brackets around the par invocation